todo_app/data/trello_items.py

```diff
@@ -64,7 +74,20 @@ def add_item(title):
 
 def save_item(item):
     itemId = item['id']
+    _, done_list_id = get_list_ids()
     r = trello_put(f'cards/{itemId}', {'idList': done_list_id})
     done_card = r.json()
```

Comment:
Not relevant to module 3, but this function is very misleading. The only thing you use from `item` is the id. So you should just take an item_id parameter. (Also note the snake_case).